test/opt/test_def_use.cpp

```diff
@@ -1414,4 +1414,51 @@ INSTANTIATE_TEST_CASE_P(
       }));
 // clang-format on
 
+TEST(DefUseManagerSingleTest, GetAnnotations) {
+  // Code
+  const char* code =
+      "OpName %1 \"struct_type\" "
+      "OpName %3 \"int_type\" "
+      "OpDecorate %1 Block "
+      "OpDecorate %1 RelaxedPrecision "
```

Comment:
Just realized we have DisassembleInst() function available. Changed back to use parameterized tests in 5f94bfb